packages/core/src/shared/logger/winstonToolkitLogger.ts

```diff
@@ -44,7 +44,7 @@ export class WinstonToolkitLogger implements Logger, vscode.Disposable {
     }
 
     public enableDebugConsole(): void {
-        this.logToDebugConsole()
+        this.logToConsole()
```

Comment:
is this correct? we still have `getDebugConsoleLogger` but it doesn't appear to actually log anything to the debug console, when I invoke a local lambda:

<img width="501" alt="image" src="https://github.com/user-attachments/assets/df7059d2-f9b3-4101-b7d0-212ebd6e7a7a">

Reply:
Alternatively, can we make sure the Output window is focused? Seems like vscode aggressively focuses the Debug Console during debugging.